tekton/pipeline_run_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"reflect"
 	"strings"
 
+	"github.com/redhat-appstudio/release-service/tekton/utils"
```

Comment:
See the warning here - I guess you added this when you switched to using utils.PipelineRef directly, but in fact you can use the existing import?

Reply:
Thanks, I thought VSCode would handle that for me

Reply:
I would have expected that too :)